### PR TITLE
profiles: Update to header 260

### PR DESCRIPTION
**Pull request by ziga-lunarg**
There is one issue, the generator will complain that the struct `VkPhysicalDeviceHostImageCopyPropertiesEXT` is ill-formed, because it contains dynamic arrays

### CODE REVIEW

File: scripts/known_good.json [hunk 1]
```diff
@@ -6,15 +6,15 @@
             "sub_dir": "Vulkan-Headers",
             "build_dir": "Vulkan-Headers/build",
             "install_dir": "Vulkan-Headers/build/install",
-            "commit": "v1.3.257"
+            "commit": "v1.3.259"
```
Comment (christophe-lunarg) [inline scripts/known_good.json:9]: Is this exepected?

**Review by christophe-lunarg — APPROVED** [review]
Looks good but it looks like it's Header 259

File: scripts/known_good.json (hunk 1 above)
Comment (ziga-lunarg) [reply scripts/known_good.json]: Forgot to include this file with the latest push, fixed now

**Comment by christophe-lunarg** [comment]
Looks great!

**Pull request merged**